test/transforms.txt

```diff
@@ -304,9 +304,9 @@ assert(e === true);
 $e = true;
 try {
   throw new Ex(_new($Error, "foo"));
-} catch(Exception $e_1_) {
-  if ($e_1_ instanceof Ex) $e_1_ = $e_1_->value;
-  call($assert, _instanceof($e_1_, $Error));
+} catch(Exception $e_2_) {
+  if ($e_2_ instanceof Ex) $e_2_ = $e_2_->value;
+  call($assert, _instanceof($e_2_, $Error));
```

Comment:
Any idea why these lines changed? Is it because of the rocambole upgrade?

Not a big deal, just wondering.

Reply:
Yep, I confirmed, it's because of the new rocambole.